specification_templates/forecast.yaml

```diff
@@ -4,9 +4,9 @@ data:
   output_root: ''
 scenarios:
   worse:
-    algorithm: 'normal' # normal, mean_level_mandate_reimposition, draw_level_mandate_reimposition
+    algorithm: 'normal' # normal, draw_level_mandate_reimposition
     solver: 'RK45'
-    system: 'new_theta'
```

Comment:
Removed 'old_theta' now that 'new_theta' has been in prod for a couple of months.